waspc/cli/Command/Deps.hs

```diff
@@ -0,0 +1,27 @@
+module Command.Deps
+  ( deps,
+  )
+where
+
+import Command (Command)
+import Control.Monad.IO.Class (liftIO)
+import Generator.ServerGenerator as Server (waspNpmDeps)
```

Comment:
When we do import, we want it to either be `qualified` or we want to name what we are importing `(foo, bar)`.
Sometime we might want to do both at the same time, but that is not often the case.

However, here you don't have `qualified` - I am not sure what that means then? I think it will import everything from `Generator.ServerGenerator` directly into the namespace (which is a bad practice), and then it will also allow you to reference them by `Server.smth` due to `as Server` part but that is not useful any more since it is all already imported, and finally `(waspNpmDeps)` will import that symbol once again which is also not useful since it was already imported.

What you instead want to do here is
```hs
import qualified Generator.ServerGenerator as Server
```

Btw I wouldn't name it just Server, because when I am reading code later, I am not sure what is that, what is just Server. If I see ServerGenerator, that is much clearer then.

So I would go with
```hs
import qualified Generator.ServerGenerator as ServerGenerator
```

Reply:
Done!